Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine AS builder
+FROM node:22-alpine AS builder
```

Comment:
Ref link for vulnerability details: https://hub.docker.com/layers/library/node/18-alpine/images/sha256-929b04d7c782f04f615cf785488fed452b6569f87c73ff666ad553a7554f0006

Reply:
Should we bump the node version in package.json while we're at it and try to keep them in sync going forward?